test/bad_dtls_test.c

```diff
@@ -381,8 +382,11 @@ static int send_finished(SSL *s, BIO *rbio)
     if (!EVP_DigestFinal_ex(handshake_md, handshake_hash, NULL))
         return 0;
 
+    md_size = EVP_MD_CTX_get_size(handshake_md);
```

Comment:
Just a quick sanity check here: we're only going to call `EVP_MD_CTX_get_size()` after a successful `EVP_DigestFinal_ex()`, which means we know `handshake_md` is not `NULL` (and `EVP_DigestFinal_ex()` actually does this very check inside it). Do we still want to add all of these checks everywhere, even in test code?

Reply:
I do not think the check does any harm.